onnxruntime_extensions/pnp/_nlp.py

```diff
@@ -22,7 +22,7 @@ def create_bert_tokenizer(ctx, name, input_names, output_names, container, opera
         # Unfortunately, there's no specific accessor function on
         # transformers.BertTokenizer to query for strip_accents.
         attrs['strip_accents'] = 1 if hf_bert_tokenizer.init_kwargs.get('strip_accents') else 0
```

Comment:
can you add a check for 'strip_accents' as well?

Reply:
done!